apps/coscrad-frontend/src/components/copyright/copyright.tsx

```diff
@@ -5,7 +5,7 @@ interface CopyrightProps {
 }
 
 export const Copyright = ({ copyrightHolder }: CopyrightProps) => (
-    <Typography variant="body2" color="text.secondary">
+    <Typography color="text.secondary">
```

Comment:
Interesting.

Reply:
That's just to get the grey colour.  Still need to work on the font size...